### PR TITLE
add CAP_NET_BIND_SERVICE to coder.service

### PR DESCRIPTION
**Pull request by johnstcn**
Folks might want to run Coder on port 80/443; they'll need to add CAP_NET_BIND_SERVICE if they want to do that (or run as root, but this is definitely preferable).

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -41,6 +41,17 @@ sudo systemctl enable --now coder
 journalctl -u coder.service -b
 ```
 
+> Note: If you wish to run Coder on a privileged port (lower than 1024),
```
Comment (mafredri) [inline docs/install.md:44]:
```suggestion
> **Note:** If you wish to run Coder on a privileged port (lower than 1024),
```

**Review by mafredri — APPROVED** [review]
This is great!

File: docs/install.md [hunk 2]
```diff
@@ -41,6 +41,17 @@ sudo systemctl enable --now coder
...
+> then you will need to edit `/usr/lib/systemd/system/coder.service` and
+> make the following change:
+>
+> ```diff
+> -AmbientCapabilities=CAP_IPC_LOCK
+> +AmbientCapabilities=CAP_IPC_LOCK CAP_NET_BIND_SERVICE
+> ```
+>
+> This will allow Coder to bind to ports lower than 1024.
+
```
[the remark this answers is not in the document]
Comment (kylecarbs) [reply docs/install.md:54]: I am wrong! Let's just add this to the service file so this isn't an issue anymore!

[the remark this answers is not in the document]
Comment (johnstcn) [reply docs/install.md:54]: Sure thing! :D

**Review by kylecarbs — APPROVED** [review]

**Pull request merged**